### PR TITLE
Added method to update the cortx component resources

### PR DESCRIPTION
**Pull request by pragamj**
Signed-off-by: pragam <pragam.jain@seagate.com>

# Problem Statement
- Added method to update cortx component resources

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
[TEST-N-NODE_TestContDeployment_20220523232031.log](https://github.com/Seagate/cortx-test/files/8759741/TEST-N-NODE_TestContDeployment_20220523232031.log)
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
[collection_log.txt](https://github.com/Seagate/cortx-test/files/8759729/collection_log.txt)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by pragamj** [comment]
The execution logs contains known failure of get-object CORTX-31558, the update in solution.yaml file is working as expected

**Review by ketanarlulkar — APPROVED** [review]

**Review by pranav-sahasrabudhe — APPROVED** [review]
Approved

File: libs/prov/prov_k8s_cortx_deploy.py [hunk 1]
```diff
@@ -1957,3 +1965,55 @@ def update_res_limit_third_party(self, filepath):
                       sort_keys=False, Dumper=noalias_dumper)
             soln.close()
         return True, filepath
+
+    def update_res_limit_cortx(self, filepath):
+        """
+        This Method is used to update the resource limits for cortx services
+        file: solution.yaml file
```
Comment (sonalk0209) [inline libs/prov/prov_k8s_cortx_deploy.py:1972]: so param is filepath right?

File: libs/prov/prov_k8s_cortx_deploy.py [hunk 2]
```diff
@@ -1957,3 +1965,55 @@ def update_res_limit_third_party(self, filepath):
...
+        returns True
```
Comment (sonalk0209) [inline libs/prov/prov_k8s_cortx_deploy.py:1973]: always returns true, asking as u r returning false if response is not true from this method
also its returning filepath as well it seems

**Review by sonalk0209 — APPROVED** [review]
Approving for merge

File: libs/prov/prov_k8s_cortx_deploy.py (hunk 1 above)
Comment (pragamj) [reply libs/prov/prov_k8s_cortx_deploy.py]: yes

**Comment by pragamj** [comment]
[collection_log.log](https://github.com/Seagate/cortx-test/files/8770243/collection_log.log)

**Pull request merged**